includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -318,9 +332,11 @@ public function set_layout( $attributes ) {
 				&& '100%' === $attributes['width']
 				&& '100%' === $attributes['height']
 			) {
-				unset( $attributes['style'], $attributes['width'], $attributes['height'] );
+				unset( $attributes['style'], $styles['position'], $attributes['width'], $attributes['height'] );
+				if ( ! empty( $styles ) ) {
+					$attributes['style'] = $this->reassemble_style_string( $styles );
+				}
```

Comment:
Unrelated to this PR, but came across it while perusing this function.

Reply:
This might warrant a test, as the bug went unnoticed.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/3728/commits/bb9c7cdca5992b772630044006c1dc2cf259565f.